### DISCUSSION TITLE
FlxParticle not able to load custom sprites

### DISCUSSION THREAD

**Discussion by FrazeRIP** [HaxeFlixel/flixel#2563] [Q&A]
Hello, I was trying to follow a tutorial and learn how to use the particle system. When I was trying to attach the particle to the emitter, it seemed like it would always show the HaxeFlixel logo, even using makeGraphic(). I also tried to use makeParticles() just like the demo, but it will crash my game instead.
Here's my code to setup the particles:

				for (i in 0 ... 20)
				{
					var p = new FlxParticle();
					p.makeGraphic(32, 32, 0xFFFFFFFF);
					p.visible = false;
					_emitter.add(p);
				}

**Comment by jobf** [comment]
For using a png particle, try this (lots more options to configure but this is a start).

```
var emitter = new FlxEmitter(FlxG.width / 2, FlxG.height / 2, 200);
emitter.loadParticles("assets/particle.png");
add(emitter);
emitter.start(false, 0.01);
```

https://user-images.githubusercontent.com/22012032/170680733-74fc8453-8bd7-46f4-9247-dd8ff5a24165.mp4
